tests/security/dns_cert/scripts/dns_cert.txt

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -u
+set -o pipefail
+
+source ${REPO_ROOT}/content/en/docs/tasks/security/dns-cert/snips.sh
+
+out=$(snip_check_the_provisioning_of_dns_certificates_1 2>&1)
+# Remove trailing spaces
+out=$(echo "$out" | sed 's/[ ]*$//')
+if [[ "$out" != *"$snip_check_the_provisioning_of_dns_certificates_2"* ]]; then
+    echo "FAILED snip_check_the_provisioning_of_dns_certificates_1:$out"; exit 1
+fi
```

Comment:
We've added some built-in [verification functions](https://github.com/istio/istio.io/blob/master/tests/util/verify.sh) to the framework. I think `_verify_contains` should work here.
```suggestion
_verify_contains "$out" "$snip_check_the_provisioning_of_dns_certificates_2" "snip_check_the_provisioning_of_dns_certificates_1"
```

Reply:
Done: the PR has been revised to use _verify_contains.